frontend/cypress/integration/common/namespace.ts

```diff
@@ -1,7 +1,9 @@
 import { And } from 'cypress-cucumber-preprocessor/steps';
+import { ensureKialiFinishedLoading } from './transition';
 
-And('user selects the {string} namespace in the NamespaceSelector', (namespace: string) => {
+And('user is looking within the {string} namespace', (namespace: string) => {
```

Comment:
mmm "is looking within" sounds strange to me, even in English, I prefer to keep the verbs "selects, clicks, filters", otherwise we are going to diverge a lot in the way of writing tests.

I know there will be a lot of types of defining these, but I prefer to describe the steps that a user would do to get a namespace.

Reply:
Yeah the wording is a little rough. I can update that. I was actually trying to go for something that didn't describe the specific actions a user would have to take in order to scope kiali to a specific namespace but rather described the desired outcome that the user wanted per the conversation @israel-hdez and I had about gherkin: https://github.com/kiali/kiali/pull/5006#discussion_r859226118.

Reply:
I'm also following a non-procedural style in a new PR. See https://github.com/kiali/kiali/pull/5013/files

As I told in the previous conversation with @nrfox, I'm heavily following best-practices/suggestion in cucumber and cypress docs. But if needed we can be flexible, I think.

Reply:
The rest of the test is ok, but this one was really hard to interpret.

Let's try to not over-think too much, I guess using verbs in the "When" side is ok, and yes, in the "Then" part we can keep describing what users expect.

There are tests that are impossible to describe without normal "clicks, types, filters," etc.